xml/System.Text/Rune.xml

```diff
@@ -1166,10 +1166,10 @@ This method is very similar to <xref:System.Text.Rune.DecodeFromUtf8(System.Read
         <Parameter Name="right" Type="System.Text.Rune" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">The first value to compare.</param>
+        <param name="right">the second value to compare.</param>
+        <summary>Returns a value indicating whether a specified <see cref="T:System.Text.Rune"/> is less than or equal to another specified <see cref="T:System.Text.Rune"/>.</summary>
+        <returns><see langword="true" /> if <paramref name="left" /> is less than or equals to <paramref name="right" />; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if <paramref name="left" /> is less than or equal to <paramref name="right" />; otherwise, <see langword="false" />.</returns>
```